### PR TITLE
feature/cart

### PR DESCRIPTION
**Pull request by przspa**
# Related issue

null
# Scope of work

Add example cart with product and empty
- [x] fade transition for empty / with product cart
- [x] fade transition for removed product 
- [x] set default spacing for SfSidebar
- [x] add title to SfSidebar 
# Screenshots of visual changes
[desktop]

[mobile]
_m

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/examples/pages/cart/Cart.vue [hunk 1]
```diff
@@ -0,0 +1,464 @@
+<template>
+  <div id="cart">
+    <SfButton style="margin: 1rem" @click="isCartSidebarOpen = true"
```
Comment (filrak) [inline packages/vue/src/examples/pages/cart/Cart.vue]: remove `style` please ;)

File: packages/vue/src/examples/pages/cart/Cart.vue [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+              title="My cart"
+              class="sf-heading--no-underline sf-heading--left"
+            />
+            <button class="my-cart__close" @click="isCartSidebarOpen = false">
+              <SfIcon icon="cross" size="20px" color="#BEBFC4" />
+            </button>
+          </div>
+        </div>
+        <div class="empty-cart">
+          <svg class="empty-cart__icon">
```
Comment (filrak) [inline packages/vue/src/examples/pages/cart/Cart.vue]: don't we have it in icons.js? if we don';t just add it ;)

File: packages/vue/src/examples/pages/cart/Cart.vue (hunk 1 above)
Comment (przspa) [reply packages/vue/src/examples/pages/cart/Cart.vue]: 🙈 sorry for that

File: packages/vue/src/examples/pages/cart/Cart.vue (hunk 2 above)
Comment (przspa) [reply packages/vue/src/examples/pages/cart/Cart.vue]: but it's too complicated icon :D it's most like the image (it have mask, group, path, circle..)

Comment (filrak) [reply packages/vue/src/examples/pages/cart/Cart.vue]: Ah you're right. Maybe just ezternwlize it to svg file then (in shared package). Keeping whole svg code here is not very clean

**Comment by filrak** [comment]
Also please add fade in/out animation for removed elements

File: packages/vue/src/examples/pages/cart/Cart.vue [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+      <div class="my-cart">
+        <SfHeading
+          title="My Account"
+          class="sf-heading--left sf-heading--no-underline my-cart__title"
+        />
+        <h3 v-if="totalItems" class="my-cart__total-items">
+          Total items: {{ totalItems }}
+        </h3>
+        <div v-if="totalItems" class="collected-product-list">
+          <transition-group name="collapse-bottom" :duration="300" tag="div">
```
Comment (filrak) [inline packages/vue/src/examples/pages/cart/Cart.vue]: imho fade-in is a better option here

**Comment by przspa** [comment]
@filrak adjusted to your comments

**Pull request merged**